src/index.js

```diff
@@ -252,7 +252,7 @@ async function run(params) {
     __OW_ACTION_NAME: actionName,
   } = process.env;
 
-  const pkgName = actionName ? actionName.match(/^\/([^/]+\/)+[^/]+$/)[1] : '';
+  const pkgName = actionName ? `${actionName.split('/')[2]}/` : '';
```

Comment:
the package name should not contain a trailing slash. otherwise it's not a name, but a path-prefix or path-segment, ....

Reply:
🙄

Reply:
I'll call it `pkgPrefix` is that acceptable?